src/Server.cpp

```diff
@@ -85,7 +129,7 @@ HttpResponse Server::handleRequest(const HttpRequest& req)
         ss << res.body().length();
         res.set_status(200, "OK");
         //TODO add MIME type
-        res.set_header("Content-Type", "text/plain");
+        res.set_header("Content-Type", "text/html");
```

Comment:
This should be solved in master, check this when you merge.

Reply:
This was a lined added to make other pages work , not what I was working on, so it can just be deleted